app/views/paypal_accounts/_ask_paypal_billing_agreement.haml

```diff
@@ -1,14 +1,18 @@
+- content_for :javascript do
+  ST.initializeNewPayPalAccountHandler("ask_billing_agreement","#{ask_billing_agreement_person_paypal_account_path}");
+  ST.initializeNewPayPalAccountHandler("ask_paypal_permissions", "#{ask_order_permission_person_paypal_account_path}");
+
 .row.paypal-connected-email-row
   %h3.paypal-account-connected
     =t("paypal_accounts.paypal_account_email_connected").html_safe
   .paypal-connected-email.paypal-account-connected
     = t("paypal_accounts.connected_account", :email => paypal_account_email)
-  .paypal-connected-change
-    = link_to t("paypal_accounts.change_account"), change_url
+  .paypal-connected-change.send-button-wrapper
+    = content_tag(:button, t("paypal_accounts.change_account"), type: :button, id: "ask_paypal_permissions")
```

Comment:
`content_tag` is new to me. Haven't seen it before.

What's the advantage vs using?:

```
%button#ask_paypal_permissions
  = t("paypal_accounts.change_account")
```